[CF1] override code explanation (#20131)
cloudflare/cloudflare-docs: src/content/docs/cloudflare-one/faq/troubleshooting.mdx

## I am getting an `Error 401: deleted_client - The OAuth Client was deleted` authorization error.

<Render file="access/error-401" product="cloudflare-one" />

## I entered an override code for WARP that was supposed to be valid for 3 hours but the override code expired faster than I expected.

[Admin override](/cloudflare-one/connections/connect-devices/warp/configure-warp/warp-settings/#admin-override) codes are time-sensitive and adhere to fixed-hour time blocks. Override codes can be reused until the end of their timeout. An override code's timeout begins in the hour the override code was generated in. Refer to the following scenarios.

> ### Scenario one: Admin generates an override code at 9:00 AM with a timeout of one hour.

Reviewer: ```suggestion
### Scenario one: Admin generates an override code at 9:00 AM with a timeout of 1 hour.
```

> If admin generates an override code with a timeout of one hour at **9:00 AM** and the user inputs the override code in their device at **9:59 AM**, the user will be able to toggle WARP on and off until **10:59 AM** (a one hour duration.)

Reviewer: ```suggestion
If admin generates an override code with a timeout of one hour at **9:00 AM** and the user inputs the override code in their device at **9:59 AM**, the user will be able to toggle WARP on and off until **10:59 AM** (a one hour duration).
```

^ Please ignore if you think we should use a different convention :D

> However, if the user attempts to enter the override code at **10:00 AM**, the override code will not work. The override code will not work because the override code was generated at **9:00 AM** and its one hour validity was counted as used in the 9:00 AM to 10:00 AM hour.

Reviewer: ```suggestion
However, if the user attempts to enter the override code at **10:00 AM**, the override code will not work. The override code will not work because the override code was generated at **9:00 AM** and its 1 hour validity was counted as used in the 9:00 AM to 10:00 AM hour.
```

> ### Scenario two: Admin generates an override code at 9:30 AM with timeout of three hours.

Reviewer: ```suggestion
### Scenario two: Admin generates an override code at 9:30 AM with timeout of 3 hours.
```

> If admin generates an override code with a timeout of three hours at **9:30 AM** and the user inputs the override code in their device at **9:59 AM**, the user will be able to toggle WARP on and off until **12:59 PM** (a three hour duration.)

Reviewer: ```suggestion
If admin generates an override code with a timeout of 3 hours at **9:30 AM** and the user inputs the override code in their device at **9:59 AM**, the user will be able to toggle WARP on and off until **12:59 PM** (a 3 hour duration.)
```

> However, if the user attempts to enter the override code at **10:00 AM**, the override code will only be valid until **12:00 PM** (a two hour duration). The override code was generated at **9:30 AM** and one hour of its total three hour validity was counted as used in the 9:00 AM to 10:00 AM hour.

Reviewer: ```suggestion
However, if the user attempts to enter the override code at **10:00 AM**, the override code will only be valid until **12:00 PM** (a 2 hour duration). The override code was generated at **9:30 AM** and 1 hour of its total 3 hour validity was counted as used in the 9:00 AM to 10:00 AM hour.
```

### Scenariot three: Admin generates an override code at 12:30 PM with a timeout of 24 hours.

> If admin generates an override code with a timeout of 24 hours at **12:30 PM** and the user inputs the override code in their device at **12:59 PM** the same day, the user will be able to toggle WARP on and off until **12:59 PM** the next day (a 24 hour duration.)

Reviewer: ```suggestion
If admin generates an override code with a timeout of 24 hours at **12:30 PM** and the user inputs the override code in their device at **12:59 PM** the same day, the user will be able to toggle WARP on and off until **12:59 PM** the next day (a 24 hour duration).
```

> However, if the user attempts to enter the override code at **1:00 PM** the same day, the override code will only be valid until **11:00 AM** the next day (a 23 hour duration). The override code was generated at **12:30 PM** and one hour of its total 24 hour validity was counted as used in the 12:00 PM to 1:00 PM hour.

Reviewer: ```suggestion
However, if the user attempts to enter the override code at **1:00 PM** the same day, the override code will only be valid until **11:00 AM** the next day (a 23 hour duration). The override code was generated at **12:30 PM** and 1 hour of its total 24 hour validity was counted as used in the 12:00 PM to 1:00 PM hour.
```
